app/src/main/java/com/recommend_tour/MainActivity.kt

```diff
@@ -1,20 +1,45 @@
 package com.recommend_tour
 
+import android.content.BroadcastReceiver
+import android.content.Context
 import android.content.Intent
+import android.content.IntentFilter
 import android.os.Bundle
+import androidx.activity.viewModels
 import androidx.appcompat.app.AppCompatActivity
-import com.recommend_tour.service.ApiService
+import com.recommend_tour.service.TourDataUpdateService
+import com.recommend_tour.viewmodels.TourListViewModel
+import dagger.hilt.android.AndroidEntryPoint
 
+@AndroidEntryPoint
+class MainActivity : AppCompatActivity() {
 
-//@AndroidEntryPoint
-class MyActivity : AppCompatActivity() {
-//    private val tourService: TourService by viewModels()
+//    private val viewModel: TourListViewModel by viewModels()
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_test)
+        setContentView(R.layout.intro_layout)
 
-        val serviceIntent = Intent(this, ApiService::class.java)
+        val serviceIntent = Intent(this, TourDataUpdateService::class.java)
```

Comment:
```suggestion
        val serviceIntent = Intent(this, TourDataUpdateService::class.java)
```
해당 service의 lifecycle은 어떻게될까요?
원하시는 기능에 대해 좀 더 효과적인 구현은 없을까요?